src/components/inputBook.js

```diff
@@ -36,6 +36,7 @@ const InputBook = () => {
   return (
     <>
       <form className="form">
+        <div className="vertical-line" />
         <h1 className="heading">ADD NEW BOOK</h1>
         <div className="inputs">
           <input type="text" value={newBook} onChange={handleChange} placeholder="Book title" className="input" required />
```

Comment:
- [ ] As  required for the project, kindly make sure your styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/) e.g.: Text colors, Fonts(size, weight, family,...), background-colors, alignment, spscing(psdding & mergin),...
### Template design
![Screenshot from 2023-04-07 03-07-49](https://user-images.githubusercontent.com/83042742/230560649-73df5c33-ccac-4b27-a6e0-c1355b46caa5.png)
### Your design
![Screenshot from 2023-04-07 03-08-29](https://user-images.githubusercontent.com/83042742/230560687-308cd956-6d53-418a-a360-9b62067caa5b.png)

Reply:
Ok, let me do the changes